src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator {
+    private double radius;
+    private double area;
+    private Color color;
+
+    public Circle(double radius) {
+        this.radius = radius;
+        this.area = getArea();
+        this.color = super.getColor();
+    }
+
+    public Circle() {
+        this.radius = 10;
+        this.area = getArea();
```

Comment:
you can't call this method inside of the constructor, this logic should be called inside of method draw()